includes/sanitizers/class-amp-meta-sanitizer.php

```diff
@@ -295,7 +295,7 @@ protected function create_charset_element() {
 			$this->dom,
 			Tag::META,
 			[
-				Attribute::CHARSET => Document\Encoding::AMP,
+				Attribute::CHARSET => Encoding::AMP,
```

Comment:
Aside: It seems this is now dead code given that it is handled by `\AmpProject\Dom\Document::insertMissingCharset()`

Reply:
I wondered why only PHPStan noticed this error, but not the unit tests... Now I know.